README.md

```diff
@@ -166,6 +167,15 @@ ShareCare:
   pinterest: true
 ```
 
+#### LinkedIn CMS intergration
```

Comment:
This section can be removed, because I think we can just enable this all the time